domain/domain.go

```diff
@@ -259,11 +256,19 @@ func NewDomain(store kv.Storage, lease time.Duration) (d *Domain, err error) {
 	}
 
 	d.infoHandle = infoschema.NewHandle(d.store)
+
 	d.ddl = ddl.NewDDL(d.store, d.infoHandle, &ddlCallback{do: d}, lease)
 	d.mustReload()
 
 	variable.RegisterStatistics(d)
 
+	// Only when the store is local that the lease value is 0.
+	// Set reload lease != 0 to ensure that loadSchemaInLoop can work normally.
+	// And if the store is local, reload lease is larger than ddl lease is ok.
```

Comment:
// Local store allows reload lease greater than ddl lease.
// For other stores, reload lease is always equal to ddl lease. ?